.circleci/config.yml

```diff
@@ -0,0 +1,115 @@
+####################
+# CircleCI configuration reference:
+#   https://circleci.com/docs/2.0/configuration-reference
+####################
+# CircleCI built-in environment variables:
+#   https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables
+####################
+
+
+####################
+# Templates: see "anchors" in https://learnxinyminutes.com/docs/yaml/
+####################
+
+save_cache_settings: &save_cache_settings
+  key: python_mozetl-{{ .Branch }}-{{ .Environment.CIRCLE_WORKFLOW_ID }}
+  paths:
+    - ~/python_mozetl/.tox
+
+prep_cache_settings: &prep_cache_settings
+  name: Clean directories to restore
+  command: |
+    : # no-op
+
+restore_cache_settings: &restore_cache_settings
+  keys:
+    - python_mozetl-{{ .Branch }}-{{ .Environment.CIRCLE_WORKFLOW_ID }}
+    - python_mozetl-{{ .Branch }}
+    - python_mozetl-
+
+common_settings: &common_settings
+  docker:
+    - image: python:2.7
+  working_directory: ~/python_mozetl
+
+# CircleCI generally only triggers builds on commits to the main repository,
+# so PRs coming from branches of the main repo simply reference builds for existing commits
+# (CIRCLE_BRANCH=branchname and CIRCLE_PR_NUMBER is unset);
+# The only time PRs will trigger a build is when the PR is referencing a fork
+# (CIRCLE_BRANCH=pull/XXX and CIRCLE_PR_NUMBER=XXX).
+early_return_for_forked_pull_requests: &early_return_for_forked_pull_requests
```

Comment:
Looks like we can remove this boilerplate, since we don't rely on any secret creds here and can treat forked PRs exactly the same as PRs from branches.